src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -352,9 +352,8 @@ and critical-path CSS or JavaScript.
 <link rel="preload" href="font.woff2" as="font" type="font/woff2" crossorigin>
 ```
 
-Starting in Chrome 83, [preloading can be used along with responsive
-images](/preload-responsive-images/) to combine both patterns for much faster image
-loading.
+Preloading can be used along with [responsive images](/preload-responsive-images/) to combine both
+patterns for much faster image loading.
```

Comment:
We still need to know which version. This article needs to be clear that this isn't universally available. If Yoav's article is wrong, it needs to be updated.

Reply:
Just pushed a commit to fix this